### PR TITLE
Backend creation and Database connection

### PR DESCRIPTION
**Pull request by Nitesh058Raj**
This PR Contains :
- created `messages.js`  controller 
- created `dbinit/init.sql`  for  database creation
- created `queries.js`  for exporting queries 
- Added routing in  `routes.js`

### CODE REVIEW

File: backend/dbinit/init.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
+CREATE DATABASE IF NOT EXISTS messageboard;
+
+USE messageboard;
+
+DROP TABLE IF EXISTS Messages;
+
+CREATE TABLE Messages (
+    ip VARCHAR(255) DEFAULT NULL,
```
Comment (WinayGupta777) [inline backend/dbinit/init.sql:8]: Means the `ip` would take "VINAY" as input also ?? 
Should we implement the validation at Frontend side ?
I want that only `ip` which starts with `172.16.155.x` can access the website. No other input should.

File: backend/package.json [hunk 2]
```diff
@@ -15,6 +17,11 @@
     "license": "UNLICENSED",
     "dependencies": {
         "cors": "^2.8.5",
-        "express": "^4.17.1"
-    }
-}
\ No newline at end of file
+        "dotenv": "^16.3.1",
+        "express": "^4.17.1",
+        "mysql": "^2.18.1"
```
Comment (WinayGupta777) [inline backend/package.json:22]: Will it be fine to use `mysql` or should be go with `mysql2` ??

File: backend/src/models/database.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+dotenv.config();
+
+const db = mysql.createPool({
+    host: process.env.DB_HOST || "localhost",
+    port: process.env.DB_PORT || 3306,
+    user: process.env.DB_USER || "admin",
+    password: process.env.DB_PASS || "abc",
+    database: process.env.DATABASE || "messageboard",
+    connectionLimit: process.env.DB_CONNECTION_LIMIT || 10
+});
```
Comment (WinayGupta777) [inline backend/src/models/database.js:13]: `connectionLimit` means Maximum Connection, Right ?

File: backend/dbinit/init.sql (hunk 1 above)
Comment (Nitesh058Raj) [reply backend/dbinit/init.sql:8]: I think we should implement that logic in FrontEnd

File: backend/package.json (hunk 2 above)
Comment (Nitesh058Raj) [reply backend/package.json:22]: I think we will update to mysql2 in future as its improved than mysql

File: backend/src/models/database.js (hunk 3 above)
Comment (Nitesh058Raj) [reply backend/src/models/database.js:13]: yes, at a time 10 concurrent requests(connections) can be established by backend to database.

**Pull request closed without merging**